src/main/java/org/kitteh/irc/client/library/implementation/EventListener.java

```diff
@@ -131,6 +132,7 @@ public void welcome(ClientReceiveNumericEvent event) {
     @Handler(priority = Integer.MAX_VALUE - 1)
     public void version(ClientReceiveNumericEvent event) {
         this.client.resetServerInfo();
+        boolean isTwitch = this.client.getEventManager().getRegisteredEventListeners().stream().anyMatch(listener -> (listener instanceof TwitchListener));
```

Comment:
This will cause issues if a custom event bus is used (post KICL events to external event bus) - `getRegisteredEventListeners` would not contain a `TwitchListener`.